### PR TITLE
Heroku support

### PR DESCRIPTION
**Pull request by rstacruz**
NB: if you're merging this, keep in mind that this also includes rauchg/slackin#3 and rauchg/slackin#2.

### CODE REVIEW

**Comment by rstacruz** [comment]
BTW, unlike #4, this actually works and I've tested it :)

**Comment by martinsvalin** [comment]
I confirmed that the deploy button worked.

File: Readme.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ You get:
 
 ### Server
 
-`deploy to heroku button here`
+[![Deploy](https://www.herokucdn.com/deploy/button.png)](https://heroku.com/deploy)
```
Comment (matiassingers) [inline Readme.md]: Perhaps use the `.svg` button instead?
`https://www.herokucdn.com/deploy/button.svg`


----

For those wondering about the missing `template` parameter in the deploy URL see: [*Using an implicit template*](https://devcenter.heroku.com/articles/heroku-button#using-an-implicit-template)

File: app.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Slackin",
+  "description": "",
```
Comment (matiassingers) [inline app.json]: Add a description?
The current repo description might not fit super well: *make your organization / channel public easily*

File: Readme.md (hunk 1 above)
Comment (rstacruz) [inline Readme.md]: gotcha!

File: app.json (hunk 2 above)
Comment (rstacruz) [inline app.json]: I just added *A little server that enables public access to a Slack server*, taken from the readme's own introduction.

**Comment by rstacruz** [comment]
Rebased from master

**Comment by rauchg** [comment]
Done

**Comment by rauchg** [comment]
Thanks a lot!

**Pull request closed without merging**